test/test_helper.rb

```diff
@@ -1,4 +1,5 @@
 ENV['RACK_ENV'] = 'test'
+require 'pry'
```

Comment:
This add either `pry` or `pry-byebug` depending on what's included in the `gemfile`